builder/amazon/common/run_config.go

```diff
@@ -18,11 +19,19 @@ import (
 var reShutdownBehavior = regexp.MustCompile("^(stop|terminate)$")
 
 type AmiFilterOptions struct {
-	Filters    map[*string]*string
-	Owners     []*string
```

Comment:
using `[]*string` or `map[*string]*string` makes the HCL2 parser crash; since using pointers here is sorts of a language abuse I made them non pointers.